### PR TITLE
validations using jquery

### PR DESCRIPTION
**Pull request by harshs14**
added validations on the registration form using jquery and few more minor changes
@bvishal8510 @deepanshu-jain1999 @SukhnandanMalhotra sir/ma'am pls review

### CODE REVIEW

**Review by bvishal8510 — COMMENTED** [review]
add validation for university roll number, mobile number at frontend.

File: core/models.py [hunk 1]
```diff
@@ -72,11 +72,11 @@ class Candidate(models.Model):
     name = models.CharField(max_length=100, blank=False)
     std_no_regex = RegexValidator(regex=r"^\d{7}$", message="Invalid Student Number", code="400")
     std_no = models.CharField(validators=[std_no_regex], blank=True, max_length=7, null=True)
-    university_roll_no = models.CharField(max_length=10, blank=True, unique=True, null=True)
-    email = models.EmailField(unique=True, blank=False)
+    university_roll_no = models.CharField(max_length=10, blank=False, unique=True, null=True)
```
Comment (akashkool) [inline core/models.py]: university_roll_no :  blank=False & null= true. You find that right ? 

& If you still accept null  in Django model then a null string in Django model can have any of two value i.e either NULL or Empty String. So two NULL or two Empty String as university_roll_no  violates **unique=True** .

File: core/templates/candidate/signup.html [hunk 2]
```diff
@@ -158,75 +158,238 @@ <h2 class="title">Registration Form</h2>
...
-        } else {
-            var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-            if (!re.test(email)) {
-                document.getElementById('email_err').innerHTML = "Email format invalid";
-                return false;
-            }
-        }
-        try{
+            if ( $(this).val() == "") {
+                    console.log("hi")
```
Comment (akashkool) [inline core/templates/candidate/signup.html:342]: what if it was an alert instead of console. 🗡

File: core/templates/candidate/signup.html [hunk 3]
```diff
@@ -158,75 +158,238 @@ <h2 class="title">Registration Form</h2>
...
+             else{
+                $("#id_submit").css('cursor', 'pointer');
+             }
+             });
+
+    });
+    function validateForm() {
+<!--        var name = document.getElementById('nameInput1').value;-->
+<!--        if (name == "") {-->
+<!--            document.getElementById('name_err').innerHTML = "Name cannot be empty";-->
```
Comment (akashkool) [inline core/templates/candidate/signup.html]: document.getElementById('name_err') == $("#name_err"). Try to keep your code clean by not mixing various approach to achieve same result. Keep it uniform.

File: core/models.py (hunk 1 above)
Comment (harshs14) [reply core/models.py]: sir will fix this
thanx

**Pull request closed without merging**